### DISCUSSION TITLE
App Name and URL

### DISCUSSION THREAD

**Discussion by caiohardrock** [Budibase/budibase#7336] [Help] [answered]
The option to edit name and url is disabled, any tips?
![Captura de Tela 2022-08-17 às 16 29 06](https://user-images.githubusercontent.com/47000086/185226516-9a120445-0c16-47ec-93d6-6d01429ef084.png)

**Comment by insomniacj93** [comment, marked as answer]
Hello, @caiohardrock try unpublishing the app.

**Reply by caiohardrock** [reply]
Thank you!
